docs/developer-guide/maps-configuration.md

```diff
@@ -1404,21 +1404,21 @@ MapStore specific:
 - `GroupList` defines a mapstore group list. Contains `Group` elements that describe a particular layer group:
 
 ```xml
-<ms:GroupList xmlns:ms="http://geo-solutions.it/mapstore/context">
-    <ms:Group xmlns:ms="http://geo-solutions.it/mapstore/context" id="Default" title="Default" expanded="true"/>
+<ms:GroupList xmlns:ms="https://mapstore2.geosolutionsgroup.com/mapstore/context">
+    <ms:Group xmlns:ms="https://mapstore2.geosolutionsgroup.com/mapstore/context" id="Default" title="Default" expanded="true"/>
```

Comment:
We should always use `mapstore.geosolutionsgroup.com` mapstore2 should never be used.

They are xmlns definisitions (that in fact do not extist), so the URL is not important. But if we want to remove geo-solutions.it, we want also to remove "mapstore2".

```suggestion
<ms:GroupList xmlns:ms="https://mapstore.geosolutionsgroup.com/mapstore/context">
    <ms:Group xmlns:ms="https://mapstore.geosolutionsgroup.com/mapstore/context" id="Default" title="Default" expanded="true"/>
```